assets/css/style.css

```diff
@@ -70,39 +70,44 @@ html[data-theme="dark"] {
 ---------------------------
 */
 
-::-webkit-scrollbar {
-    width: 16px;
-}
-
-::-webkit-scrollbar-track {
-    -webkit-box-shadow: inset 0 0 6px #bb86fc;
-    border-radius: 2px;
-    background: #f1f1f1;
-}
-
-::-webkit-scrollbar-thumb {
-    border-radius: 2px;
-    max-height: 10px;
-    -webkit-box-shadow: inset 0 0 6px #bb86fc;
-    background: #bb86fc;
-    transition: 0.3s;
-}
-
-::-webkit-scrollbar-thumb:hover {
-    background: #ca9eff;
-    
-}
-
-body {
-    height: 100vh;
+body,html{
     margin: 0;
-    overflow-x: hidden;
+    padding:0;
+    max-width: 100%;
+    overflow-x:hidden;
     color: var(--text);
+    overflow-y: auto;
     font-family: "Poppins", sans-serif;
     background-color: var(--bg);
+    overflow-y: auto;
 }
 
+::-webkit-scrollbar{
+    width: 15px; 
+}
 
+::-webkit-scrollbar-track {
+    background-color: rgb(204, 204, 204);
+    height:100vh;
+    box-shadow: inset 1px 2px 4px rgb(170, 170, 170);
+}
+::-webkit-scrollbar-thumb {
+    border: 1.5px solid rgba(217, 45, 252, 0.945);
+    max-height: 14px;
+    width:15px;
+    background: #d261ff;
+    box-shadow: 2px 3px 3px rgb(170, 170, 170);
+    border-radius: 150px;
+}
+::-webkit-scrollbar:hover{
+   width:17px;
+     
+  }
+ ::-webkit-scrollbar-thumb:hover{
+   background-color:#d28eff;
```

Comment:
define dedicated variables for this with an appropriate name like scroll-thumb-cl